spopt/region/base.py

```diff
@@ -7,52 +7,23 @@
 from scipy.spatial import KDTree
 
 
-class RegionMixin(object):
```

Comment:
Is `RegionMixin` going away?

Reply:
It's not used anywhere within `spopt` and it seemed like it was legacy from the days of `pysal/region`. Should we keep it? We can absolutely keep it if that would be prudent.

Reply:
I think the intention was to have a common set of methods in all the models to enforce standardization (`model.solve`). So I think we should have a discussion of whether that is still desired - if it is then we can go back and refactor existing classes to adopt that, if not, then we can merge this and do away with the Mixin.

Reply:
How about I add it back in here now and we can open an issue on whether to get rid of it the future? Probably better to have time for discussion, as you say.